collections/Bodl/MS_Bodl_738.xml

```diff
@@ -62,29 +129,29 @@
+                        <item>Feria II, Nocturnum II (Ps. 26, fol. 49va);</item>
```

Comment:
some cases here and elsewhere of extra spaces in the `ex` element e.g. `cap<ex> itu</ex>lo`